tests/test_linters.py

```diff
@@ -321,18 +309,19 @@ def test_framework_operator_no_ops_imported(tmp_path, monkeypatch, import_line):
         "from charms.reactive.stuff import Stuff",
     ],
 )
-def test_framework_reactive_used_ok(tmp_path, monkeypatch, import_line):
+@pytest.mark.parametrize("charm_name", ["foobar", "foo-bar"])
+def test_framework_reactive_used_ok(tmp_path, import_line, charm_name):
     """The reactive framework was used.
 
     Parametrized args:
     - import_line: different ways to express the import
     """
     # metadata file with needed name field
     metadata_file = tmp_path / "metadata.yaml"
-    metadata_file.write_text("name: foobar")
+    metadata_file.write_text(f"name: {charm_name}")
 
     # a Python file that imports charms.reactive
-    entrypoint = tmp_path / "reactive" / "foobar.py"
+    entrypoint = tmp_path / "reactive" / "{}.py".format(charm_name.replace("-", "_"))
```

Comment:
Don't just duplicate the tested code here, is error prone (if the original code has a difficult to see bug, we're just reproducing it here). 

Be explicit! Parametrize passing both charm and module names. Thanks!

Reply:
Fixed